test/unit/tools/test_tool_deps.py

```diff
@@ -499,23 +499,23 @@ def __test_base_path():
 
 
 def test_parse():
-    with __parse_resolvers('''<dependency_resolvers>
+    with __parse_resolvers(b'''<dependency_resolvers>
```

Comment:
I think these additions of `b` are wrong, the proper solution is changing line 634 of this file to use `tempfile.NamedTemporaryFile('w+')` to open the temporary file in text mode.

@bgruening Can you open a PR to fix it?

Reply:
Addressed in https://github.com/galaxyproject/galaxy/pull/6455 .